### DISCUSSION TITLE
Number of computers

### DISCUSSION THREAD

**Discussion by AdriaDiaz** [TheAlbertDev/MASB#40] [Q&A] [answered]
Is it possible to access/work with a Git repository from two different computers, e.g. one computer at home and one at faculty, or do I have to work with only one computer?

**Comment by Albert-Alvarez** [comment, marked as answer]
You can work from any computer. In fact, one of the advantages of Git is that it is a distributed version control system. All you need to do is, from the first computer, upload your changes to GitHub via a `git push` and then, from the second computer, grab your updated repository with a `git clone` or a `git pull` depending on whether you had cloned the repository on the second computer.

In part A of the first practice (Arduino), we see the `git push` command. In the second part, the B (STM32), we see the command `git pull`.
